README.md

```diff
@@ -24,112 +33,105 @@ This module depends on a correctly configured [GitHub Provider](https://www.terr
 Add the module to your Terraform resources like so:
 
 ```hcl
-module "foo-cli" {
-  source                                    = "github.com/withmethod/terraform-module-github-repository?ref=0.3.1"
-  name                                      = "foo-cli"
-  description                               = "foo CLI"
-  homepage_url                              = "https://withmethod.com/open-source/"
-  private                                   = true
-  has_issues                                = true
-  has_wiki                                  = false
-  allow_merge_commit                        = false
-  allow_squash_merge                        = true
-  allow_rebase_merge                        = false
-  has_downloads                             = false
-  auto_init                                 = true
-  gitignore_template                        = "Terraform"
-  license_template                          = "apache-2.0"
-  enable_team_repository                    = 1
-  team_repository_team                      = "${github_team.internal.id}"
-  team_repository_permission                = "pull"
-  branch                                    = "master"
-  enable_branch_protection                  = 1 // only works after initial creation of repository
-  enforce_admins                            = true
-  req_status_checks_strict                  = false
-  req_status_checks_context                 = ["continuous-integration/travis-ci"]
-  req_pr_reviews_dismiss_stale_reviews      = true
-  req_pr_reviews_require_code_owner_reviews = false
-  req_pr_reviews_dismissal_users            = []
-  req_pr_reviews_dismissal_teams            = []
-  restrictions_users                        = []
-  restrictions_teams                        = []
+module "simple-example" {
+  source  = "operatehappy/repository/github"
+  version = "2.0.0"
+
+  name    = "oh-demo-simple-example"
+  private = false // NOTE: private repositories require a paid subscription
```

Comment:
```suggestion
private = false
```

Private repositories are [free now](https://github.blog/2019-01-07-new-year-new-github/) in most contexts (OSS FTW) 🏆

Reply:
Oh my - you can tell how long I had this PR around 😂 - literally wrote this at the end of last year / beginning of this year and never got around to updating this.

Yes, VERY good catch. Thanks!